### PR TITLE
Fix : [FO] FILTERING - Resize screen to mobile

### LINKED ISSUE #483: [FO]  FILTERING - Resize screen to mobile
Resize screen to mobile: 

- [ ]  open filters, resize screen back to desktop. ➡️ Empty modal is open.
- [ ]  resize it back ➡️ Price slider doesn't work.

### PR DESCRIPTION
**Pull request by ga-devfront**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Fix the problem of front office about filtering resize screen to mobile.
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     |  no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/hummingbird/issues/483
| Sponsor company   | @PrestashopCorp.
| How to test?      | Test with the explination inside the ticket.

### CODE REVIEW

**Review by Hlavtox — APPROVED** [review]

**Review by hibatallahAouadni — APPROVED** [review]
Hello @ga-devfront 

LGTM, QA :heavy_check_mark: 

Thanks!

File: src/js/responsive-toggler.test.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import EVENTS from '@js/constants/events-map';
+import selectorsMap from '@constants/selectors-map';
 import initResponsiveToggler from './responsive-toggler';
 import initEmitter from './prestashop';
 import {
   desktopElement, mobileElement, mobileElementId, contentValue, desktopElementId,
 } from './constants/mocks/swapElements-data';
+import ThemeType = Theme.ThemeType;
```
Comment (SharakPL) [inline src/js/responsive-toggler.test.ts:8]: @ga-devfront why this import? seems useless.

Comment (ga-devfront) [reply src/js/responsive-toggler.test.ts:8]: on my side then I test without this it fail.

Comment (SharakPL) [reply src/js/responsive-toggler.test.ts:8]: Maybe this import is required elsewhere.
obraz

Is this even a valid way of importing modules?

Comment (SharakPL) [reply src/js/responsive-toggler.test.ts:8]: https://github.com/evanw/esbuild/issues/1283

**Pull request merged**